migrate/20250110_add_kubernetes_models.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    create_table(:kubernetes_cluster) do
```

Comment:
I strongly suggest adding a `project_id` column and filling it.
https://github.com/ubicloud/ubicloud/pull/2594

Reply:
done!